reports/model_features/renv/.gitignore

```diff
@@ -0,0 +1,7 @@
+library/
```

Comment:
Running notes for myself:

- Remove renv environment
- Make each report runnable on its own:

```r
baseline_run_id <- ifelse(exists("params"), params$baseline_run_id, "foo")
```